src/frequenz/client/electricity_trading/_client.py

```diff
@@ -49,6 +50,32 @@ class _Sentinel:
 NO_VALUE = _Sentinel()
 
 
+def validate_decimal_places(value: Decimal, decimal_places: int, name: str) -> None:
```

Comment:
I think this is a very nice function to have some unit tests.

Reply:
Indeed, I thought it would be too obvious at first but I guess it's still good to have